apps/pipe-easy/src/app/app.component.ts

```diff
@@ -1,21 +1,36 @@
 import { NgFor } from '@angular/common';
-import { Component } from '@angular/core';
+import {
+  ChangeDetectorRef,
+  Component,
+  Pipe,
+  PipeTransform,
+} from '@angular/core';
+
+@Pipe({
+  standalone: true,
+  pure: true,
+  name: 'heavyComputation',
+})
+export class HeavyComputationPipe implements PipeTransform {
+  transform(value: string, ...args: [index: number]) {
+    return `Piped@${new Date()}: ${value} - ${args[0]}`;
+  }
+}
 
 @Component({
   standalone: true,
-  imports: [NgFor],
+  imports: [NgFor, HeavyComputationPipe],
   selector: 'app-root',
   template: `
     <div *ngFor="let person of persons; let index = index">
-      {{ heavyComputation(person, index) }}
+      {{ person | heavyComputation : index }}
     </div>
   `,
 })
 export class AppComponent {
   persons = ['toto', 'jack'];
 
-  heavyComputation(name: string, index: number) {
-    // very heavy computation
-    return `${name} - ${index}`;
+  constructor(cdr: ChangeDetectorRef) {
+    setInterval(() => cdr.markForCheck(), 1000);
```

Comment:
I think this is only for testing purposes ? 🧐

Reply:
Yes, just to see that the pipe output does not change